src/emu/epoc/src/services/window/window.cpp

```diff
@@ -733,6 +734,57 @@ namespace eka2l1 {
         guest_evt_.key_evt_.repeats = 0;            // TODO?
     }
     
+    static std::array<bool, 3> mouse_button_pressed {};
```

Comment:
Static variable once modified, other modificiation will be invalid (on MSVC that's it, and on GCC its event more buggy). I highly suggest to rename this to button_pressed and move this into window_server along with the other function.